Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -2091,13 +2091,38 @@ void GridFormattingContext::layout_absolutely_positioned_element(Box const& box)
     GridItem item { box, {}, {}, {}, {} };
     if (!is_auto_row) {
         auto row_placement_position = resolve_grid_position(box, GridDimension::Row);
-        item.row = row_placement_position.start;
-        item.row_span = row_placement_position.span;
+
+        // https://www.w3.org/TR/css-grid-2/#abspos
+        // 10.1. With a Grid Container as Containing Block
+        // If a grid-placement property refers to a non-existent line either by
+        // explicitly specifying such a line or by spanning outside of the existing
+        // implicit grid, it is instead treated as specifying auto (instead of
+        // creating new implicit grid lines).
+
+        int max_row = m_grid_rows.size();
+
+        // If the row start refers to a line before the first row or after the
```

Comment:
it's a bit confusing that those comments are a mix of spec text and author interpretation of how this works. could we add a `NOTE:` prefix for everything that is not a spec text?